aspnetcore/fundamentals/openapi/samples/9.x/GetDocument.Insider/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System.Reflection;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOpenApi();
+
+if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider")
+{
+   // 'IServiceCollection' does not contain a definition for 'AddDefaults' 
+    builder.Services.AddDefaults();
```

Comment:
I think this is the fix:
```suggestion
    builder.AddServiceDefaults();
```